tasks/copy_to_glacier/README.md

```diff
@@ -322,177 +322,4 @@ An example of a message is shown below:
-                                                                    Must match value in storage_class table.
-                ORCA_DEFAULT_BUCKET (str): Name of the default S3 Glacier
-                                                     ORCA bucket files should be
-                                                     archived to.
-                METADATA_DB_QUEUE_URL (string, required): SQS URL of the metadata queue.
-        
-        Args:
-            event: Event passed into the step from the aws workflow.
-                See schemas/input.json and schemas/config.json for more information.
-        
-        
-            context: An object required by AWS Lambda. Unused.
-        
-        Returns:
-            The result of the cumulus task. See schemas/output.json for more information.
-    
-    should_exclude_files_type(file_key: str, exclude_file_types: List[str]) -> bool
-        Tests whether file is included in {excludeFileTypes} from copy to glacier.
-        Args:
-            file_key: The key of the file within the s3 bucket.
-            exclude_file_types: List of extensions to exclude in the backup.
-        Returns:
-            True if file should be excluded from copy, False otherwise.
-    
-    task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str, Any]
-        Copies the files in {event}['input']
-        to the ORCA glacier bucket defined in ORCA_DEFAULT_BUCKET.
-        
-        Environment Variables:
-            ORCA_DEFAULT_BUCKET (string, required): Name of the default ORCA S3 Glacier bucket.
-                Overridden by bucket specified in config.
-            DEFAULT_MULTIPART_CHUNKSIZE_MB (int, optional): The default maximum size of chunks to use when copying.
-                Can be overridden by collection config.
-            METADATA_DB_QUEUE_URL (string, required): SQS URL of the metadata queue.
-        
-        Args:
-            event: Passed through from {handler}
-            context: An object required by AWS Lambda. Unused.
-        
-        Returns:
-            A dict representing input and copied files. See schemas/output.json for more information.
-
-DATA
-    Any = typing.Any
-    CONFIG_EXCLUDE_FILE_TYPES_KEY = 'excludeFileTypes'
-    CONFIG_MULTIPART_CHUNKSIZE_MB_KEY = 's3MultipartChunksizeMb'
-    CONFIG_ORCA_DEFAULT_BUCKET_OVERRIDE_KEY = 'orcaDefaultBucketOverride'
-    CONFIG_ORCA_DEFAULT_STORAGE_CLASS_OVERRIDE_KEY = 'orcaDefaultStorageCl...
-    Dict = typing.Dict
-    FILE_BUCKET_KEY = 'bucket'
-    FILE_FILEPATH_KEY = 'key'
-    FILE_HASH_KEY = 'checksum'
-    FILE_HASH_TYPE_KEY = 'checksumType'
-    LOGGER = <cumulus_logger.CumulusLogger object>
-    List = typing.List
-    MB = 1048576
-    OS_ENVIRON_DEFAULT_STORAGE_CLASS_KEY = 'DEFAULT_STORAGE_CLASS'
-    OS_ENVIRON_ORCA_DEFAULT_BUCKET_KEY = 'ORCA_DEFAULT_BUCKET'
-    Union = typing.Union
-```
-
-## pydoc sqs_library.py
```

Comment:
I think ## pydoc sqs_library.py should stay

Reply:
Added to api scripts here and elsewhere.